target/linux/ramips/mt76x8/base-files/etc/board.d/02_network

```diff
@@ -101,6 +101,10 @@ ramips_setup_interfaces()
 		ucidef_add_switch "switch0" \
 			"1:lan:4" "2:lan:3" "3:lan:2" "4:lan:1" "0:wan" "6@eth0"
 		;;
+	joowin,jw-wr758ac,v[12])
```

Comment:
Ditto. And please separate this into two lines.

Reply:
Fixed in latest push.